{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -5,23 +5,24 @@ description = "{{cookiecutter.project_description}}"
 authors = [{ name = "{{cookiecutter.author}}", email = "{{cookiecutter.email}}" }]
 readme = "README.md"
 keywords = ['python']
-requires-python = ">=3.9,<4.0"
+{%- set minor_version = cookiecutter.main_python_version.split('.')|map('int')|list %}
+requires-python = ">={{cookiecutter.main_python_version}},<4.0"
```

Comment:
Still should remain 3.9; not the main_python_version acts as a minimum python version. A good Python package should aim to support all [currently supported Python versions](https://devguide.python.org/versions/).

At least, a package should start with this. If for some reason some versions need to be dropped, e.g. because some dependencies only support >=3.10, then it makes sense to update this manually.

Reply:
Okay, I had no idea what you meant earlier by main python version. I thought it was supposed to be the minimum version. So, it should be ">=3.9,<3.13" if I select 3.12 as `main_python_version`, right?

Reply:
> Still should remain 3.9; not the main_python_version acts as a minimum python version. A good Python package should aim to support all [currently supported Python versions](https://devguide.python.org/versions/).

I get it, I add all supported versions in my packages. I just thought this version selection would be an interesting addition.